<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Sportsed::Client::Admin::CompetitionEditWidget</class>
 <widget class="QWidget" name="Sportsed::Client::Admin::CompetitionEditWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>754</width>
    <height>552</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Name:</string>
       </property>
       <property name="buddy">
        <cstring>nameEdit</cstring>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="nameEdit"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Sport:</string>
       </property>
       <property name="buddy">
        <cstring>sportBox</cstring>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QComboBox" name="sportBox">
       <property name="editable">
        <bool>true</bool>
       </property>
       <item>
        <property name="text">
         <string>Orienteering</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Running</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Skiing</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Biathlon</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="1" rowspan="2">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Clients</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLineEdit" name="clientSearchEdit">
        <property name="placeholderText">
         <string>Search...</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTreeView" name="clientsView"/>
      </item>
      <item>
       <widget class="QPushButton" name="clientDisconnectBtn">
        <property name="text">
         <string>Disconnect</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="changeClientPasswordBtn">
        <property name="text">
         <string>Change Password</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Stages</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,0,0">
        <item>
         <widget class="QComboBox" name="stageBox"/>
        </item>
        <item>
         <widget class="QPushButton" name="stageAddBtn">
          <property name="text">
           <string>Add</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="stageRemoveBtn">
          <property name="text">
           <string>Remove</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QWidget" name="stageWidget" native="true">
        <layout class="QFormLayout" name="formLayout_2">
         <item row="0" column="0">
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Name:</string>
           </property>
           <property name="buddy">
            <cstring>stageNameEdit</cstring>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="stageNameEdit"/>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>Type:</string>
           </property>
           <property name="buddy">
            <cstring>stageTypeEdit</cstring>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QComboBox" name="stageTypeEdit"/>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>Date:</string>
           </property>
           <property name="buddy">
            <cstring>stageDateEdit</cstring>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QDateTimeEdit" name="stageDateEdit"/>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>Discipline:</string>
           </property>
           <property name="buddy">
            <cstring>stageDisciplineBox</cstring>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QComboBox" name="stageDisciplineBox">
           <property name="editable">
            <bool>true</bool>
           </property>
           <item>
            <property name="text">
             <string>Sprint</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Middle</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Long</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="QCheckBox" name="stageTotalesBox">
           <property name="text">
            <string>Included in total result</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
